src/util/matcher/stitcher.js

```diff
@@ -0,0 +1,104 @@
+function stitcher(processedPoints, overlay, map) { // jshint ignore:line
+  var center, corners;
+  var sectors = {s00: [], s01: [], s10: [], s11: [], population: []};
+  for (var i in processedPoints.points) {
+    sectors.s00[i] = [];
+    sectors.s01[i] = [];
+    sectors.s10[i] = [];
+    sectors.s11[i] = [];
+    center = processedPoints.images[i].getCenter();
+    corners = processedPoints.images[i].getCorners();
+    for (var j in processedPoints.points[i]) {
+      if (
+        processedPoints.points[i][j].lng > corners[1].lng &&
+        processedPoints.points[i][j].lng < center.lng
+      ) {
+        if (
+          processedPoints.points[i][j].lat > corners[3].lat &&
+          processedPoints.points[i][j].lat < center.lat
+        ) {
+          sectors.s10[i].push(processedPoints.points[i][j]);
+        } else {
+          sectors.s00[i].push(processedPoints.points[i][j]);
+        }
+      } else {
+        if (
+          processedPoints.points[i][j].lat > corners[2].lat &&
+          processedPoints.points[i][j].lat < center.lat
+        ) {
+          sectors.s11[i].push(processedPoints.points[i][j]);
+        } else {
+          sectors.s01[i].push(processedPoints.points[i][j]);
+        }
+      }
+    }
+  }
+  for (i=0; i<processedPoints.points.length; i++) {
+    sectors.population[i] = [];
+    sectors.population[i].push([
+      sectors.s00[i].length,
+      sectors.s01[i].length,
+      sectors.s10[i].length,
+      sectors.s11[i].length
+    ]);
+  }
+  var max = 0,
+    min = 10,
+    max_ = 0,
+    max_idx;
+  for (i in sectors.population[0][0]) {
+    if (sectors.population[0][0][i] > max) {
+      max = sectors.population[0][0][i];
+      max_idx = i;
+    }
+    if (sectors.population[0][0][i] < min) {
+      min = sectors.population[0][0][i];
+    }
+  }
+  if (max !== min) {
+    var coordinates = sectors[Object.keys(sectors)[max_idx]][0];
+    for (var u in processedPoints.points[0]) {
+      for (var v in coordinates) {
+        if (processedPoints.points[0][u] === coordinates[v]) {
+          if (processedPoints.confidences[0][u] > max_) {
+            max_ = processedPoints.confidences[0][u];
+          }
+        }
+      }
+    }
+    var best_point =
+      processedPoints.points[0][processedPoints.confidences[0].indexOf(max_)];
+    var corresponding_best_point =
+      processedPoints.points[1][processedPoints.confidences[0].indexOf(max_)];
+    document.querySelector(
+      "#map > div.leaflet-pane.leaflet-map-pane > div.leaflet-pane.leaflet-marker-pane"
```

Comment:
Ah, we should probably not use `#map` here - different leaflet maps may have different IDs, and also, what if there is more than one Leaflet map per page? We should base this off of the Leaflet map object container!